ItemDescription3/tableEXT/salesInvoiceLineTabExt.50402.TabExt.al

```diff
@@ -0,0 +1,13 @@
+tableextension 50402 appsalesInvoiceLineTabExt extends "Sales Invoice Line"
+{
+    fields
+    {
+        field(50400; "appDescription 3"; Text[100])
+        {
+            DataClassification = CustomerContent;
+            caption = 'Description 3';
+            TableRelation = IF (Type = CONST(Item)) Item."appDescription 3" WHERE(Blocked = CONST(false),
```

Comment:
@Ams272 Same question about table relation. I think it's not required to be here.

Reply:
modified